etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/CopyOrcOrderProviderToObrOrderProviderTest.groovy

```diff
@@ -0,0 +1,195 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom
+
+import gov.hhs.cdc.trustedintermediary.ExamplesHelper
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirResource
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetadata
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.Practitioner
+import org.hl7.fhir.r4.model.ServiceRequest
+import spock.lang.Specification
+
+
+class CopyOrcOrderProviderToObrOrderProviderTest extends Specification{
+
+    final String PRACTITIONER_EXTENSION_URL = "https://reportstream.cdc.gov/fhir/StructureDefinition/xcn-practitioner"
+
+    def transformClass
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+        TestApplicationContext.register(MetricMetadata, Mock(MetricMetadata))
+        TestApplicationContext.injectRegisteredImplementations()
+
+        transformClass = new CopyOrcOrderProviderToObrOrderProvider()
+    }
+
+    def "when both practitioner resources populated"() {
```

Comment:
Should we be naming these test differently so that the outcome can be inferred more accurately? 
So this test would become something like: 
`def "when both fields are populated, ORC.12 is copied over (OBR.16)"`